mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -130,15 +130,22 @@ private void configureExtension(MavenSession session, Configuration configuratio
         final MavenProjectConfigurator mavenProjectConfigurator = new MavenProjectConfigurator(configuration);
         final File dumpedConfigFile = configuration.dump(Paths.get("").toFile());
         session.getAllProjects().forEach(mavenProject -> {
-            mavenProjectConfigurator.configureTestRunner(mavenProject.getModel());
-            copyConfigurationFile(mavenProject.getModel(), dumpedConfigFile);
-            if (isFailedStrategyUsed()) {
-                SurefireReportStorage.copySurefireReports(mavenProject.getModel());
+            String skipTests = mavenProject.getProperties().getProperty("skipTests");
```

Comment:
How are we covering following scenario with above code?

```xml
<plugin>
    <artifactId>maven-surefire-plugin</artifactId>
    <configuration>
      <skipTests>true</skipTests>  
    </configuration>
</plugin>
```

Reply:
Yeah and don't forget that it is also possible to use just `skip`:
```xml
    <configuration>
      <skip>true</skip>  
    </configuration>
```

Reply:
Yeah and keep in mind that it is also possible to use just `skip`:
```xml
    <configuration>
      <skip>true</skip>  
    </configuration>
```